modules/ossm-running-v2-v3-multitenant-deployment-model.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-v2-v3-side-by-side-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-running-v2-v3-multitenant-deployment-model_{context}"]
+= Running {SMProductName} 2.6 and {SMProductName} 3 using multi-tenant deployment model
```

Comment:
Do you think this would be allowed to make it a teeny bit shorter?
```suggestion
= Running {SMProductName} 2.6 and {SMProduct} 3 using multi-tenant deployment model
```